client/src/components/Panel.tsx

```diff
@@ -68,6 +69,14 @@ export const Panel = ({
 					</IconButton>
 				</Tooltip>
 
+				<Tooltip title="home">
+					<IconButton >
+						<a href="home">
+							<img src={HomeIcon} alt="home avatar" className="panel-avatar" />
+						</a>
```

Comment:
In react-router apps for local routes we need to use the Link component when redirecting users, or `useHistory` and `history.push`.
For absolute routes we use `<a href`

Good explanation here https://stackoverflow.com/a/61028649/13395599